docs/firmware/device-sdk/light-db/concepts-lightdb.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Concepts - LightDB

```

Comment:
These are fine to leave for now but would like them to drop the "Concepts - " and be more descriptive, like "Sending logs."